### ISSUE TITLE
Exception in CSV plugin when using with remote SSH host

### ISSUE THREAD

**Issue by xmcp** [SeeSharpSoft/intellij-csv-validator#279]
I use PyCharm on Windows with an SSH remote Linux interpreter & deployment.

It turns out that CSV plugin will crash in background (I did't open CSV files, it just crashes silently, leaving a exclamation mark on the bottom-right of IDE window ↓) under this scenario.
![image](https://user-images.githubusercontent.com/6646473/115953674-6d39b600-a51f-11eb-83d0-5b406185549a.png)

The traceback shows that PyCharm seems to use some UUID-like stuff to represent path names on the remote host, triggering an `InvalidPathException` of this plugin.

Traceback:
```
java.nio.file.InvalidPathException: Illegal char <<> at index 0: <835d7ae1-4344-4666-bc29-31fb457b610e>\raid\...path.names.on.remote.host...\...file.name....json
	at java.base/sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
	at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
	at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
	at java.base/sun.nio.fs.WindowsPath.parse(WindowsPath.java:92)
	at java.base/sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:229)
	at java.base/java.nio.file.Path.of(Path.java:147)
	at java.base/java.nio.file.Paths.get(Paths.java:69)
	at net.seesharpsoft.intellij.plugins.csv.CsvStorageHelper.getFilePath(CsvStorageHelper.java:37)
	at net.seesharpsoft.intellij.plugins.csv.CsvStorageHelper.getFileInProject(CsvStorageHelper.java:42)
	at net.seesharpsoft.intellij.plugins.csv.components.CsvFileAttributes.lambda$cleanupAttributeMap$0(CsvFileAttributes.java:62)
	at java.base/java.util.HashMap.forEach(HashMap.java:1336)
	at net.seesharpsoft.intellij.plugins.csv.components.CsvFileAttributes.cleanupAttributeMap(CsvFileAttributes.java:61)
	at net.seesharpsoft.intellij.plugins.csv.CsvPlugin$1.run(CsvPlugin.java:66)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:998)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsync$5(CoreProgressManager.java:497)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:228)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:178)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:688)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:634)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:64)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:165)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:228)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

Environment:
```
PyCharm 2021.1 (Professional Edition)
Build #PY-211.6693.115, built on April 6, 2021
(license information removed)
Runtime version: 11.0.10+9-b1341.35 amd64
VM: Dynamic Code Evolution 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 2032M
Cores: 8
Registry: search.everywhere.settings=true
Non-Bundled Plugins: Statistic (4.1.7), net.seesharpsoft.intellij.plugins.csv (2.16.2), mobi.hsz.idea.gitignore (4.1.0), org.jetbrains.plugins.vue (211.6693.111)
```

**Comment by SeeSharpSoft** [comment]
Never thought about that possibility - thank you for reporting!

**Comment by SeeSharpSoft** [comment]
This issue is fixed with the latest plugin version 2.16.3. Feel free to reopen if the error still persists in this version.
